vllm/model_executor/models/gpt_oss.py

```diff
@@ -174,6 +176,11 @@ def __init__(
             has_bias=True,
             activation="swigluoai",
             is_sequence_parallel=self.is_sequence_parallel,
+            custom_routing_function=(
+                gpt_oss_custom_routing_function
+                if not current_platform.is_rocm()
```

Comment:
nit: just check cuda to help other platforms
```suggestion
                if current_platform.is_cuda()
```